app/services/handover_date_service.rb

```diff
@@ -29,7 +29,7 @@ def self.handover(raw_offender)
                                    start_date: nil, handover_date: nil,
                                    reason: :parole_mappa_2_3
 
-      elsif offender.target_hearing_date < offender.hearing_outcome_received + 12.months
+      elsif offender.next_thd.present? && offender.next_thd < offender.last_hearing_outcome_received + 12.months
```

Comment:
Be careful with this line. I would put brackets around the left and right portions (either side of the <) and check it still works. In the console I got: 
```
irb(main):005:0> nil.present? && nil < 4+1
=> false
irb(main):006:0> (nil.present? && nil) < (4+1)
Traceback (most recent call last):
        1: from (irb):6
NoMethodError (undefined method `<' for false:FalseClass)
```

Reply:
I'm not sure I understand the purpose of putting the brackets
Both sides of the && are treated as separate expressions, which is what we want in this scenario. The behaviour seen in your first example is what we're after, short circuiting and escaping a NoMethodError if `next_thd` is nil.

Reply:
Yes, precisely!